cart.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">

```

Comment:
```suggestion
<html lang="ko">

```
해당 속성은 현재 페이지 속의 텍스트가 어떤 '언어'로 작성되었는지 나타내어 주는 속성입니다. 요걸 한국어로 설정해주시면 보조 수단을 통해 웹을 이용하시는 분들이 웹을 조금 더 편하게 이용하실 수 있는 웹 접근성 관련 속성입니다! 프론트는 요런 걸 챙겨주면 좋아용